dotnet3.5/S3/CreateBucket/CreateBucket.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CreateBucket]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using System;
+using System.Threading.Tasks;
+
+namespace CreateBucket
+{
+    public class CreateBucket
+    {
+        // Specify your bucket region (an example region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the new bucket
+        private const string NEW_BUCKET_NAME = "*** provide the name of the bucket with source object ***";
+
+        static void Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+            Console.WriteLine($"\nCreating a new bucket, named: {NEW_BUCKET_NAME}.");
+
+            CreatingBucketAsync(_s3Client, NEW_BUCKET_NAME).Wait();
+
+            // If FindBucketLocationAsyn doesn't return a value, the creation failed.
+            var bucketLocation = FindBucketLocationAsync(_s3Client, NEW_BUCKET_NAME);
+
+            if(!String.IsNullOrEmpty(bucketLocation.ToString()))
+            {
+                Console.WriteLine($"\n\nBucket {NEW_BUCKET_NAME} created at {bucketLocation}.");
+            } else
+            {
+                Console.WriteLine("Could not create bucket.");
```

Comment:
Might be nice to include troubleshooting information to the end of this string to help the user. Do we know why the bucket couldn't be created?

Reply:
Unless there's an error, no we don't know. I suppose I could make some suggestions in the comments.